go/control/api/api.go

```diff
@@ -48,9 +52,12 @@ type Status struct {
 }
 
 // Shutdownable is an interface the node presents for shutting itself down.
+// TODO: Rename Shutdownable to Node? And Node to NodeImpl?
```

Comment:
Could also rename this interface to `ControlledNode` and leave the `Node` as-is.